docs/setup/local.md

```diff
@@ -8,32 +8,71 @@ Deploying KubeEdge locally is used to test, never use this way in production env
 
 ## Setup Cloud Side (KubeEdge Master Node)
 
+### Create CDRs
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/devices/devices_v1alpha1_device.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/devices/devices_v1alpha1_devicemodel.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/reliablesyncs/cluster_objectsync_v1alpha1.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/reliablesyncs/objectsync_v1alpha1.yaml
+```
+
+
 ### Prepare config file
 
 ```shell
 # cloudcore --minconfig > cloudcore.yaml
```

Comment:
I remember there is a step to build those binaries, but I cannot find them anymore, where is it?

Reply:
I think build process should not place here, build by hand or download release or `go get` should be documented in one doc.

Reply:
Do we have one already? We use to have but I have no idea where it it now.

Reply:
We have it, https://github.com/kubeedge/kubeedge/blob/master/docs/setup/build.md.

Reply:
This is not what I mean, it's purely for cross-compiling.

Reply:
Get it, it would be better to place in build.md

Reply:
Actually, we use to have it but I have no idea where they are gone.

Reply:
I know we used to have it.

Reply:
BTW, I still think we need the doc on the manually build.